### DISCUSSION TITLE
What is the recommended way to use Nuxt Content with Nuxt Image

### DISCUSSION THREAD

**Discussion by remiconnesson** [nuxt/content#2082] [Q&A]
What is the recommended way to use Nuxt Content with Nuxt Image ?

I plan to build using `nuxi generate` and to put my images in the `/static` directory

**Comment by LoganTann** [comment]
You can override the default ProseImg component to use nuxt-img instead of the native img tag

EDIT : see the following comment https://github.com/nuxt/content/discussions/2082#discussioncomment-6105683, my code below is not sufficient.

<details><summary>Original code</summary>
<p>


```html
<!-- components/content/ProseImg.vue -->
<template>
  <nuxt-img />
</template>
```

</p>
</details>

**Comment by nobkd** [comment]
The example in this discussion (https://github.com/nuxt/content/discussions/2082#discussioncomment-6104227) lacks, as the attributes are not transferred to the underlying component.

See this solution: https://github.com/nuxt/content/issues/390#issuecomment-1344593577 to [override](https://content.nuxtjs.org/api/components/prose) the Content `img` tag.
But instead of using commas (`,`) to separate the properties, I'd use spaces (` `) instead, as not using spaces can lead to props not being parsed properly.
Also, the path can also be `components/content/**/ProseImg.vue` instead.

If you don't want to override the default `img` for whatever reason, just name your component differently. You can the use `:your-nuxt-img-component-name{alt="" yourprops}` in your markdown instead. As you can see, you can't take advantage of the default `img` Markdown syntax in that case.
